### ISSUE TITLE
Assets url error in prod

### ISSUE THREAD

**Issue by YLMC2017** [farm-fe/farm#795]
### Steps to reproduce

![k1](https://github.com/farm-fe/farm/assets/35885819/39c098da-72db-4e7f-8829-7fce8ec00ae0)

![k2](https://github.com/farm-fe/farm/assets/35885819/b1101d12-8f42-4f42-a52a-f966d71cf3c1)

https://github.com/YLMC2017/my-vue-app-2/tree/assets_test

npm run build
npm run preview

### What is expected?

pic loaded

![k3](https://github.com/farm-fe/farm/assets/35885819/1679a57f-20e8-43ff-b105-2356fdc4ef0c)

### What is actually happening?

pic load fail

### System Info

```Shell
System:
    OS: Windows 10 10.0.19044
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
    Memory: 1.10 GB / 15.79 GB
  Binaries:
    Node: 20.10.0 - C:\Program Files\nodejs\node.EXE
    npm: 10.2.3 - C:\Program Files\nodejs\npm.CMD
    pnpm: 8.10.3 - ~\AppData\Roaming\npm\pnpm.CMD
  Browsers:
    Edge: Spartan (44.19041.3636.0), Chromium (119.0.2151.72)
    Internet Explorer: 11.0.19041.3636
  npmPackages:
    @farmfe/core: ^0.14.7 => 0.14.7
```

**Comment by wre232114** [comment]
it's fixed

**Comment by wre232114** [comment]
fixed in 0.14.10
